server/meshmodel/kubernetes/relationships/hierarchical_inventory.json

```diff
@@ -381,6 +381,70 @@
           }
         ]
       }
+    },
+    {
+      "allow": {
+        "from": [
+          {
+            "kind": "Deployment",
+            "model": "kubernetes",
+            "patch": {
+              "patchStrategy": "replace",
+              "mutatorRef": [
```

Comment:
@MUzairS15 Please check paths.

Reply:
@RipulHandoo why? What's wrong with the paths? What aren't you sure about? Why do you need this aspect of your work double-checked?

Reply:
@leecalcote In my local setup, dropping a lease inside a deployment isn't updating the configuration; it might be related to paths. And double-checking isn't wrong, I guess?

Reply:
Double-checking is the right thing to do.

I'm asking in order to try to 1) understand whether there is unexpected behavior occurring (I didn't see mention of this), 2) understand whether you're doing your homework or just relying upon reviewers to check your work, 3) identify what we might improve in our development process to enhance our tooling / the flow, so that testing is easier (but, I wasn't sure why you're asking for extra review).

Make sense?

Reply:
This information was missing, hence my question: " In my local setup, dropping a lease inside a deployment isn't updating the configuration; it might be related to paths."